spec/providers/coreIntegration/rtcpeerconnection.integration.src.js

```diff
@@ -54,8 +59,28 @@ module.exports = function (pc, dc, setup) {
     }, function (err) {
       console.error('RTC failed: ',err);
     });
-  });
+  };
   
+  it("Sends string messages across data channels", function (done) {
+    sendMessageToPeer('message from alice', function(result) {
+      expect(result.text).toEqual('message from alice');
+      done();
```

Comment:
probably should call done() after closing the channels, i think that would just be moving the callback(msg) after the bob.close() above